Allow to not fail the `use-profiles` action when profile(s) can't be applied to a project(s) (#80)
codemagic-ci-cd/cli-tools: CHANGELOG.md

Version 0.5.1
-------------

**Improvements**

> - Feature: add the `warn-only` flag to `xcode-project use-profiles` to not fail the action when profile(s) can't be applied to an Xcode project(s)

Reviewer: ```suggestion
- Feature: add the `warn-only` flag to `xcode-project use-profiles` not to fail the action when profiles can't be applied to any of the Xcode projects
```
